core/src/main/resources/schema/app/20220323113700_CB-16557_Add_DNS_resolver.sql

```diff
@@ -0,0 +1,9 @@
+-- // CB-16557 Check if all unbound related node list config are removed without using salt
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE stack ADD COLUMN IF NOT EXISTS domainDnsResolver VARCHAR(255) DEFAULT 'UNKNOWN';
```

Comment:
I don't know the pg version on every environment but it's safer to create without the default in case of large tables, then add the default
see: https://dba.stackexchange.com/a/216163

Reply:
I think I will leave old stacks with null since it is a valid value